tests/storage/export.go

```diff
@@ -914,6 +914,46 @@ var _ = SIGDescribe("Export", func() {
 		Expect(*export.Status.TokenSecretRef).ToNot(BeEmpty())
 	})
 
+	It("Should honor TTL by cleaning up the the VMExport altogether", func() {
+		sc, exists := libstorage.GetRWOFileSystemStorageClass()
+		if !exists {
+			Skip("Skip test when Filesystem storage is not present")
+		}
+
+		pvc, _ := populateKubeVirtContent(sc, k8sv1.PersistentVolumeFilesystem)
+		ttl := 2 * time.Minute
+		export := &exportv1.VirtualMachineExport{
```

Comment:
I wont make it a must for this PR, but there are a couple of functions and this code which created an export object, all with small differences like adding toke, ttl, without secreat, etc..  It is better to unit the code as much as possible instead of continue duplicating it.